Sources/AST/Environment/Environment+Type.swift

```diff
@@ -82,7 +91,9 @@ extension Environment {
   }
 
   // The type of a range.
-  public func type(ofRangeExpression rangeExpression: RangeExpression, enclosingType: RawTypeIdentifier, scopeContext: ScopeContext) -> RawType {
+  public func type(ofRangeExpression rangeExpression: RangeExpression,
+                   enclosingType: RawTypeIdentifier,
+                   scopeContext: ScopeContext) -> RawType {
```

Comment:
Do we still need the ofRangeExpression because the types should be sufficient?

Reply:
Perhaps but I think this is outside of the scope of making trivial changes to satisfy the linter

Reply:
https://swift.org/documentation/api-design-guidelines/#argument-labels

Reply:
Again this was more that the trivial changes in the PR suggest that there is a systematic problem as opposed to a minor one. I have no issue with merging this as is.